src/main/java/org/folio/innreach/domain/service/impl/CirculationServiceImpl.java

```diff
@@ -198,6 +201,19 @@ public InnReachResponseDTO cancelItemHold(String trackingId, String centralCode,
     return success();
   }
 
+  @Override
+  public InnReachResponseDTO itemReceived(String trackingId, String centralCode, ItemReceivedDTO itemReceivedDTO) {
+    var transaction = getTransaction(trackingId, centralCode);
+
+    if (transaction.getState() != ITEM_SHIPPED){
+      throw new IllegalArgumentException("Item is not shipped.");
+    }
```

Comment:
lets unify the message for invalid transaction state. There are couple of examples in other methods:
`    Assert.isTrue(transaction.getState() == ITEM_SHIPPED,
      "Unexpected transaction state: " + transaction.getState());`